Models/Comment.cs

```diff
@@ -1,10 +1,9 @@
 using System;
-using System.Collections.Generic;
 using System.Text.Json.Serialization;
 
 namespace Karma.Models
 {
-	public class Comment: Entity
+    public class Comment: Entity
```

Comment:
Maybe we should delete Comment and Category classes because I don't think we are going to use them anymore.

Reply:
Also Category repository could be deleted.